app/views/companies/new.html.erb

```diff
@@ -1,20 +1,61 @@
-<div class="signup_form">
-  <h2>Start hiring</h2>
-  <h4>Whether you’re an employer, hiring manager, or community leader, all we need is an interest from you in learning
-    more about TechHire, and a passion to find and hire talented employees in your community who are ready to work.</h4>
-    <%= form_for @company do |f| %>
-      <%= render "signup_form", f: f %>
-      <div class="form-group">
-        <%= f.check_box :hiring %>
-        <%= f.label(:hiring, "I am ready to commit to hiring") %>
+<div class="mod-heading check">
+  <h1>Sign Up</h1>
+</div>
+
+<div class="layout-wrap">
+  <%= form_for @company, url: companies_path do |f| %>
+    <p class="intro">
+      Whether you’re an employer, hiring manager, or community leader, all we need is an interest from you in learning more about TechHire, and a passion to find and hire talented employees in your community who are ready to work.
+    </p>
+
+    <div class="row">
+      <%= f.label(:name, "Name (First and Last)") %>
+      <%= f.text_field :name, required: true %>
+    </div>
+
+    <div class="row">
+      <%= f.label(:organization, "Organization") %>
+      <%= f.text_field :organization, required: true  %>
+    </div>
+
+    <div class="row">
+      <%= f.label(:title, "Title") %>
+      <%= f.text_field :title, required: true  %>
+    </div>
+
+    <div class="row cols-25-75">
+      <div class="col">
+        <%= f.label(:state, "State") %>
+        <%= f.select :state, options_for_select(state_helper, @company.state) %>
       </div>
-      <div class="form-group">
-        <%= f.number_field :hire_count, in: 0..500, step: 1, value: 0 %><br>
-        <%= f.label(:hire_count, "people through TechHire") %>
+
+      <div class="col">
+        <%= f.label(:city, "City") %>
+        <% cache states do %>
```

Comment:
this was previously caching on `@cities`, which is nil, so it would just never cache.